.github/workflows/nodejs.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
-        node-version: [14.x, 16.x, 18.x]
+        node-version: [14.x, 16.x, 18.x, 19.x]
```

Comment:
Would it make sense to add this version to windows matrix too, or is this unix specific?

Reply:
No, you’re right, we should be doing this :+1: